deploy/bootstrap.py

```diff
@@ -497,7 +524,7 @@ def _build_conda_env(options, mpi, activate_base):
     if conda_mpi == 'nompi':
         mpi_prefix = 'nompi'
     else:
-        mpi_prefix = f'mpi_{mpi}'
+        mpi_prefix = conda_mpi
```

Comment:
Almost, but this needs to be:
```suggestion
        mpi_prefix = f'mpi_{conda_mpi}'
```

Reply:
My explanation wasn't very good!